repository/character.go

```diff
@@ -0,0 +1,39 @@
+package repository
+
+import (
+	"context"
+	"log"
+
+	"github.com/TanZng/toh-api/models"
+)
+
+type CharacterRepository interface {
```

Comment:
Interface implementations should behold into a struct and refactor by constructor NewCharacterRepository